kotlin/http4k/gradle.properties

```diff
@@ -0,0 +1,4 @@
+kotlin_version=1.4.10
+shadow_version=5.0.0
```

Comment:
could we use `+` instead of `5.0.0`

imho, we do not really need to pin `shadow`

Reply:
fair enough - was taking the model from the ktor example, so you might want to change that one as well. 😄

Reply:
We need to find a way to run all `kotlin` **frameworks** with the same config, perhaps sharing the same config (at least some parts)